x-pack/plugins/siem/public/management/pages/index.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo } from 'react';
+import { Redirect, Route, Switch } from 'react-router-dom';
+import { SpyRoute } from '../../common/utils/route/spy_routes';
+import { PolicyContainer } from './policy';
+import {
+  MANAGEMENT_ROUTING_ENDPOINTS_PATH,
+  MANAGEMENT_ROUTING_POLICIES_PATH,
+  MANAGEMENT_ROUTING_ROOT_PATH,
+} from '../common/constants';
+import { ManagementPageView } from '../components/management_page_view';
+import { NotFoundPage } from '../../app/404';
+
+const TmpEndpoints = () => {
```

Comment:
Thoughts on adding this in with the next PR which would display the Endpoints?  Could avoid another "placeholder" page in the near term

Reply:
Yes, this will be removed in the next PR, where I will move (and rename?) the endpoint-hosts over to the management section.

Reply:
I should expand a little more: The default landing view is the `Endpoints`, thus I wanted to minimize the amount of "temporary" code, so I included the Endpoints sub-tab/route, but am currently display this temporary view. So my next PR will remove this one component here and replace it with the real - so less code to refactor 😃

Reply:
Got it, thanks for the explanation